### DISCUSSION TITLE
Blind color option ? / eye protection option ?

### DISCUSSION THREAD

**Discussion by gaikoz11** [ppy/osu#20917] [Ideas] [answered]
Eye protection option . I think its good option for those playing for hours or for those players playing only in evening . 

Color blind option . Is for player have a blind color . 

I know this is unnecessary idea but im just sharing it .

**Comment by nagi-desuuu** [comment]
There are skins that already do this, plus some beatmaps have only one custom color set to a really dark color, maybe black.
Still, you can achieve some degree of success by setting background dim to around 90% if both of those options aren't your cup of tea.

**Comment by bdach** [comment, marked as answer]
1. #17058
2. What would "eye protection mode" even do?

**Reply by SirHonkkk** [reply]
same thing as combo color normalization i suppose.

**Comment by localcake** [comment]
If by "eye protection mode" you mean blue light filter then that comes default with Windows 10 and with some monitors. I would also recommend just taking frequent breaks from using your PC.
